tests/parsers/agilent_gen5/testdata/absorbance/240411_172731_BNCH2345883_abs450_96well_non_numeric_values.json

```diff
@@ -40,21 +35,20 @@
                             }
                         }
                     ],
+                    "measurement time": "2024-04-11T17:27:15+00:00",
+                    "plate well count": {
+                        "value": 83,
```

Comment:
This is correct, even when a value doesn't have a valid measurement, plate_well_count should not change.

Reply:
@joshua-benchling agree with Nate's comment here. We should obtain count of all the wells even if no actual measurement from some.